lib/zendesk_api/middleware/response/logger.rb

```diff
@@ -18,7 +18,10 @@ def call(env)
           @logger.debug dump_debug(env, :request_headers)
 
           @app.call(env).on_complete do |env|
-            @logger.info("Status #{env[:status].to_s}")
+            info = "Status #{env[:status].to_s}"
```

Comment:
don't need the `.to_s` here ... it's automatic

Reply:
fixed in db1e458